code/iaas/model/src/main/java/io/cattle/platform/core/util/SystemLabels.java

```diff
@@ -14,7 +15,7 @@
     public static final String LABEL_RANCHER_UUID = "io.rancher.container.uuid";
     public static final String LABEL_RANCHER_NETWORK = "io.rancher.container.network";
     public static final String LABEL_DISPLAY_NAME = "io.rancher.container.display_name";
-    public static final String LABEL_HEALTHCHECK_SKIP = "io.rancher.host.healthcheck.skip";
+    public static final String LABEL_HEALTHCHECK_DEPLOY = "io.rancher.infra_service.healthcheck.deploy";
```

Comment:
I don't like the change to infra_services and we have sort-of pattern of using the word "strategy" at the end. Can you change it to
`io.rancher.host.healthcheck.deploy_strategy`?

Let me know if you disagree and have a good reason for switching it to infra_services

Reply:
I got the point now. This will be refer to https://github.com/rancher/agent/blob/v1.6/core/hostInfo/os_c.go#L57:3 . I will push a PR to set label `io.rancher.host.healthcheck.deploy_strategy` to `skip`